### PR TITLE
update values on review page

### PR DESCRIPTION
**Pull request by katedee**
Updating the values as requested, except for RRSP which is being handled separately and is not guaranteed for review.

localhost_3005_review (2)

### CODE REVIEW

File: api/user.json [hunk 1]
```diff
@@ -154,32 +148,30 @@
     "climateActionIncentiveIsRural": null
   },
   "refund": {
-    "Total Payable": {
```
Comment (sboisvert) [inline api/user.json]: IS changing this going to break something somewhere? It doesn't seem used, but I really don't know :)

Comment (katedee) [reply api/user.json]: nope! I added it just for this page for now

File: locales/en.json [hunk 2]
```diff
@@ -255,5 +255,9 @@
   "Total Deducted (437)": "Total Deducted (437)",
   "Total Credits (482)": "Total Credits (482)",
   "Total Refund (484)": "Total Refund (484)",
-  "I confirm that, as far as I know, all the information in this tax return is accurate": "I confirm that, as far as I know, all the information in this tax return is accurate"
+  "I confirm that, as far as I know, all the information in this tax return is accurate": "I confirm that, as far as I know, all the information in this tax return is accurate",
+  "Ontario trillium benefits": "Ontario trillium benefits",
+  "Detailed breakdown of tax refund": "Detailed breakdown of tax refund",
+  "Medical Expenses (330)": "Medical Expenses (330)",
+  "Net Ontario tax (428)": "Net Ontario tax (428)"
 }
```
Comment (pcraig3) [inline locales/en.json]: Can you remove this? Let's avoid changing these files while both Cayce and Elise are working on them.

File: api/user.json [hunk 3]
```diff
@@ -154,32 +148,30 @@
...
   },
   "benefits": {
-    "Canada child benefit": 6639.00,
-    "Climate action incentive": 0.00,
-    "GST/HSTC": 0.00,
-    "Trillium": 300.00,
-    "totalBenefits": 6939.00
+    "Climate action incentive": 154.00,
+    "GST/HSTC": 440.00,
+    "Ontario trillium benefits": 307.92,
```
Comment (pcraig3) [inline api/user.json]: I think the correct words are "Ontario Trillium Benefit"

https://www.ontario.ca/page/ontario-trillium-benefit

File: api/user.json [hunk 4]
```diff
@@ -154,32 +148,30 @@
...
+      "amount": 719.88
     }
   },
   "benefits": {
-    "Canada child benefit": 6639.00,
-    "Climate action incentive": 0.00,
-    "GST/HSTC": 0.00,
-    "Trillium": 300.00,
-    "totalBenefits": 6939.00
+    "Climate action incentive": 154.00,
```
Comment (pcraig3) [inline api/user.json]: This one is right 👍 

https://www.canada.ca/en/revenue-agency/services/tax/individuals/topics/about-your-tax-return/tax-return/completing-a-tax-return/deductions-credits-expenses/line-449-climate-action-incentive.html

**Pull request merged**